notify/src/wait/wfi/LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
`wait/wfi` seems to be some package. is it possible to add it as a package? Storing it here looks wired